src/components/accordion/_accordion.scss

```diff
@@ -70,15 +74,24 @@
     // This is styled to look like a link not a button
     .govuk-accordion__open-all {
       @include govuk-font($size: 16);
-      @include govuk-link-common;
-      display: inline;
+      position: relative;
+      z-index: 1;
+      margin: 0;
+      margin-right: 5px;
+      padding: 0;
       border-width: 0;
       color: $govuk-link-colour;
       background: none;
       cursor: pointer;
+      -webkit-appearance: none;
+
+      @include govuk-link-common;
+      @include govuk-link-style-default;
 
-      &:focus {
-        background: none;
+      // Remove default button focus outline in Firefox
+      &::-moz-focus-inner {
```

Comment:
Good catch – wonder if we should list this bug fix (and the similar fix for section headings) separately in the changelog?